enterprise/backend/test/metabase_enterprise/sandbox/api/user_test.clj

```diff
@@ -31,17 +31,16 @@
                     (map :email))))
         (testing "... even if they are a group manager"
           (mt/with-premium-features #{:advanced-permissions :sandboxes}
-            (mt/with-group [group {:name "a group"}]
```

Comment:
Fixed this test to just use the `&group` anaphor in the test assertions instead of an additional group (this was failing due to the new group starting out with full perms, and thus overriding the sandbox)